coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,167 @@
+exts = {'.w': {'C'},
+        '.c': {'C'},
+        '.cs': {'C#'},
+        '.csx': {'C#'},
+        '.c++': {'C++'},
+        '.cpp': {'C++'},
+        '.cxx': {'C++'},
+        '.h': {'C', 'C++'},
+        '.h++': {'C++'},
+        '.hh': {'C++'},
+        '.hpp': {'C++'},
+        '.hxx': {'C++'},
+        '.tcc': {'C++'},
+        '.tpp': {'C++'},
+        '.cmake': {'CMake'},
+        '.cmake.in': {'CMake'},
+        '.css': {'CSS'},
+        '._coffee': {'CoffeeScript'},
+        '.coffee': {'CoffeeScript'},
+        '.cson': {'CoffeeScript'},
+        '.iced': {'CoffeeScript'},
+        '.cu': {'Cuda'},
+        '.cuh': {'Cuda'},
+        '.dart': {'Dart'},
+        '.f': {'FORTRAN'},
+        '.f03': {'FORTRAN'},
+        '.f08': {'FORTRAN'},
+        '.f77': {'FORTRAN'},
+        '.f90': {'FORTRAN'},
+        '.f95': {'FORTRAN'},
+        '.for': {'FORTRAN'},
+        '.fpp': {'FORTRAN'},
+        '.go': {'Go'},
+        '.htm': {'HTML'},
+        '.html': {'HTML'},
+        '.xhtml': {'HTML'},
+        '.hs': {'Haskell'},
+        '.hsc': {'Haskell'},
+        '.json': {'JSON'},
+        '.java': {'Java'},
+        '._js': {'JavaScript'},
+        '.bones': {'JavaScript'},
+        '.jake': {'JavaScript'},
+        '.js': {'JavaScript'},
+        '.jsfl': {'JavaScript'},
+        '.jsm': {'JavaScript'},
+        '.jss': {'JavaScript'},
+        '.jsx': {'JavaScript'},
+        '.pac': {'JavaScript'},
+        '.sjs': {'JavaScript'},
+        '.ssjs': {'JavaScript'},
+        '.jl': {'Julia'},
+        '.lua': {'Lua'},
+        '.nse': {'Lua'},
+        '.rbxs': {'Lua'},
+        '.markdown': {'Markdown'},
+        '.md': {'Markdown'},
+        '.mkd': {'Markdown'},
+        '.mkdown': {'Markdown'},
+        '.ron': {'Markdown'},
+        '.matlab': {'Matlab', 'Octave'},
+        '.m': {'Objective-C', 'Matlab'},
+        '.mm': {'Objective-C'},
+        '.cl': {'OpenCL'},
+        '.opencl': {'OpenCL'},
+        '.aw': {'PHP'},
+        '.ctp': {'PHP'},
+        '.php': {'PHP'},
+        '.php3': {'PHP'},
+        '.php4': {'PHP'},
+        '.php5': {'PHP'},
+        '.phpt': {'PHP'},
+        '.nqp': {'Perl'},
+        '.perl': {'Perl'},
+        '.ph': {'Perl'},
+        '.pl': {'Perl'},
+        '.plx': {'Perl'},
+        'pm': {'Perl'},
```

Comment:
I think `.` is okay, otherwise it's not an extension. We could automatically prepend it, but then special file names like `CMakeLists.txt` can't be taken into account any more --> I'm for keeping `.`